tests/README.md

```diff
@@ -8,44 +8,120 @@ Priority is to provide very simple system of test scenarios, easily extendable
 so everyone contributing remediation can also provide cases where this
 remediation works.
 
-## Prerequisites
+The Test Suite executes the scans, remediations, and test scenarios on VMs or containers.\
+To test content using VMs, *Libvirt backend* is used;\
+to test content using containers, either *Podman* or *Docker* backend can be used.
 
-You can use the more powerful VM-based tests, or more lightweight container-based tests.
+Once a domain or container is prepared it can be used indefinitely.\
+The Test Suite can perform tests focused on a profile or a specific rule.\
+Typically, for a test run, a sequence of scan, remediation and scan is peformed.
 
-For the Test Suite to work, you need to have libvirt domains prepared for
-testing.
-SSG Test Suite currently does not provide automated provisioning of domains.
-You can use kickstart usable for Red Hat Enterprise Linux 7 (and of course
-CentOS 7) and Fedora in `kickstarts` directory, which installs machine capable of
-running tests with SSG Test Suite.
+## How to prepare a backend for testing
+For the Test Suite to work, you need to have a backend image prepared for testing.\
+You can use a powerful full-blown VM backend, or a lightweight container backend.
 
-If you want to use your own domain, make sure `openscap-1.2.15` and
-`qemu-guest-agent` are installed there, `root` is accessible via ssh and that
-packages packages can be installed (for RHEL7, it means subscription enabled).
-For testing Ansible remediations, it is sufficient to have `root` accessible via
-ssh on the libvirt domain and have Ansible installed on the host machine.
+SSG Test Suite currently does not provide automated provisioning of VMs or containers.
 
-### Domain preparation
+### Libvirt backend
 
+To use Libvirt backend, you need to have:
 
-1. Install domain, using provided kickstart
+- These packages installed in your host
```

Comment:
+  `python2-libvirt`/`python3-libvirt` (for the test suite to be able to work with the libvirt backend), `openssh-clients` (`ssh` and `scp` on the host side) and `libvirt-client` (optional, provides `virsh`)

Reply:
Should be resolved.